docs/configuration/index.md

```diff
@@ -106,7 +106,7 @@ There are four JVM parameters that we set on all of our services:
      - The temp directory should not be volatile tmpfs.
      - This directory should also have good read and write speed.
      - Avoid NFS mount.
-     - The `org.apache.druid.java.util.metrics.SysMonitor` requires execute privileges on files in `java.io.tmpdir`. If you are using the system monitor, do not set `java.io.tmpdir` to `noexec`.
+     - The `org.apache.druid.java.util.metrics.OshiSysMonitor` and `org.apache.druid.java.util.metrics.SysMonitor` both require execute privileges on files in `java.io.tmpdir`. If you are using the system monitor, do not set `java.io.tmpdir` to `noexec`. Note that SysMonitor is deprecated. We recommend using OshiSysMonitor.
```

Comment:
@hardikbajaj - do you know if the OshiSysMonitor needs this privilege?

Reply:
@hardikbajaj ping